main.star

```diff
@@ -287,25 +294,28 @@ def run(plan, args={}):
             )
             plan.print("Succesfully light-beaconchain-explorer")
         elif additional_service == "prometheus_grafana":
```

Comment:
ah i see there's an ordering issue - can we do the following; remove this if condition completely from inside the loop

and do something like this at the end; perhaps we could also add an `or

```
if "prometheus_grafana" in args_with_defaults.additional_services or "beacon_metrics_gazer" in args_with_defaults.additional_services:
     launch_it_like_we_do
```

Reply:
Okay, as you suggested, I will remove the condition and add the check only at the end. I think there is no need to add the `or` part since Prometheus can run independently of the `beacon_metrics_gazer`.